<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>BrushColor</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BrushColorButton" name="toolButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Bruch size</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="BrushSizeButton" name="toolButton_2">
        <property name="minimumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="GLwidget" name="openGLWidget">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuImport_Texture">
     <property name="title">
      <string>Import Texture</string>
     </property>
     <addaction name="actionAlbedo"/>
     <addaction name="actionMetallic"/>
     <addaction name="actionRoughness"/>
     <addaction name="actionAmbient_Occlusion"/>
     <addaction name="actionEmissive"/>
     <addaction name="actionDisplacement"/>
    </widget>
    <widget class="QMenu" name="menuSave_Texture">
     <property name="title">
      <string>Save Texture</string>
     </property>
     <addaction name="actionAlbedo_Save"/>
     <addaction name="actionMetallic_Save"/>
     <addaction name="actionRoughness_Save"/>
     <addaction name="actionAmbient_Occlusion_Save"/>
     <addaction name="actionEmissive_Save"/>
     <addaction name="actionDisplacement_Save"/>
    </widget>
    <addaction name="actionImport_models"/>
    <addaction name="menuImport_Texture"/>
    <addaction name="menuSave_Texture"/>
    <addaction name="actionSave_all_Texture"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionCenter_Camera"/>
    <addaction name="separator"/>
    <addaction name="actionToggle_Wireframe"/>
    <addaction name="separator"/>
    <addaction name="actionDefault_View"/>
    <addaction name="actionTexture_View"/>
    <addaction name="actionUV_View"/>
    <addaction name="actionRender_View"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <widget class="QMenu" name="menuActive_Texture">
     <property name="title">
      <string>Active Texture</string>
     </property>
     <addaction name="actionAlbedo_Texture"/>
     <addaction name="actionMetallic_Texture"/>
     <addaction name="actionRoughness_Texture"/>
     <addaction name="actionAmbient_Occlusion_Texture"/>
     <addaction name="actionEmissive_Texture"/>
     <addaction name="actionDisplacement_Texture"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="actionClear_Active_Texture"/>
    <addaction name="actionSet_Paint"/>
    <addaction name="actionSet_Stroke_Width"/>
    <addaction name="menuActive_Texture"/>
    <addaction name="actionClear_All_Textures"/>
    <addaction name="actionSet_Emissive_Color"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbou_Program"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuView"/>
   <addaction name="menuEdit"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionImport_models"/>
   <addaction name="actionSave_all_Texture"/>
   <addaction name="separator"/>
   <addaction name="actionCenter_Camera"/>
   <addaction name="actionClear_Active_Texture"/>
   <addaction name="actionClear_All_Textures"/>
   <addaction name="actionToggle_Wireframe"/>
   <addaction name="actionSet_Emissive_Color"/>
   <addaction name="separator"/>
   <addaction name="actionDefault_View"/>
   <addaction name="actionTexture_View"/>
   <addaction name="actionUV_View"/>
   <addaction name="actionRender_View"/>
   <addaction name="separator"/>
  </widget>
  <action name="actionImport_models">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/OpenFolder.png</normaloff>:/menu/menuBarIco/OpenFolder.png</iconset>
   </property>
   <property name="text">
    <string>Import models</string>
   </property>
  </action>
  <action name="actionCenter_Camera">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/cameraFocus.png</normaloff>:/menu/menuBarIco/cameraFocus.png</iconset>
   </property>
   <property name="text">
    <string>Center Camera</string>
   </property>
  </action>
  <action name="actionClear_Active_Texture">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/DeleteActiveTexture.png</normaloff>:/menu/menuBarIco/DeleteActiveTexture.png</iconset>
   </property>
   <property name="text">
    <string>Clear Active Texture</string>
   </property>
  </action>
  <action name="actionSet_Paint">
   <property name="text">
    <string>Set Paint</string>
   </property>
  </action>
  <action name="actionSet_Stroke_Width">
   <property name="text">
    <string>Set Stroke Width</string>
   </property>
  </action>
  <action name="actionToggle_Wireframe">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/wireframe.png</normaloff>:/menu/menuBarIco/wireframe.png</iconset>
   </property>
   <property name="text">
    <string>Toggle Wireframe</string>
   </property>
  </action>
  <action name="actionDefault_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/Default.png</normaloff>:/menu/menuBarIco/Default.png</iconset>
   </property>
   <property name="text">
    <string>Default View</string>
   </property>
  </action>
  <action name="actionTexture_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/Texture.png</normaloff>:/menu/menuBarIco/Texture.png</iconset>
   </property>
   <property name="text">
    <string>Texture View</string>
   </property>
  </action>
  <action name="actionAbou_Program">
   <property name="text">
    <string>About Program</string>
   </property>
  </action>
  <action name="actionUV_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/UV.png</normaloff>:/menu/menuBarIco/UV.png</iconset>
   </property>
   <property name="text">
    <string>UV View</string>
   </property>
  </action>
  <action name="actionRender_View">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/Render.png</normaloff>:/menu/menuBarIco/Render.png</iconset>
   </property>
   <property name="text">
    <string>Render View</string>
   </property>
  </action>
  <action name="actionAlbedo_Texture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Albedo Texture</string>
   </property>
  </action>
  <action name="actionMetallic_Texture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Metallic Texture</string>
   </property>
  </action>
  <action name="actionRoughness_Texture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Roughness Texture</string>
   </property>
  </action>
  <action name="actionClear_All_Textures">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/deleteAllTexture.png</normaloff>:/menu/menuBarIco/deleteAllTexture.png</iconset>
   </property>
   <property name="text">
    <string>Clear All Textures</string>
   </property>
  </action>
  <action name="actionSet_Emissive_Color">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/setpaint.png</normaloff>:/menu/menuBarIco/setpaint.png</iconset>
   </property>
   <property name="text">
    <string>Set Emissive Color</string>
   </property>
  </action>
  <action name="actionAmbient_Occlusion_Texture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Ambient Occlusion Texture</string>
   </property>
  </action>
  <action name="actionEmissive_Texture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Emissive Texture</string>
   </property>
  </action>
  <action name="actionDisplacement_Texture">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Displacement Texture</string>
   </property>
  </action>
  <action name="actionAlbedo">
   <property name="text">
    <string>Albedo</string>
   </property>
  </action>
  <action name="actionMetallic">
   <property name="text">
    <string>Metallic</string>
   </property>
  </action>
  <action name="actionRoughness">
   <property name="text">
    <string>Roughness</string>
   </property>
  </action>
  <action name="actionAmbient_Occlusion">
   <property name="text">
    <string>Ambient Occlusion</string>
   </property>
  </action>
  <action name="actionEmissive">
   <property name="text">
    <string>Emissive</string>
   </property>
  </action>
  <action name="actionDisplacement">
   <property name="text">
    <string>Displacement</string>
   </property>
  </action>
  <action name="actionSave_all_Texture">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/menu/menuBarIco/SaveFile.png</normaloff>:/menu/menuBarIco/SaveFile.png</iconset>
   </property>
   <property name="text">
    <string>Save all Texture</string>
   </property>
  </action>
  <action name="actionAlbedo_Save">
   <property name="text">
    <string>Albedo</string>
   </property>
  </action>
  <action name="actionMetallic_Save">
   <property name="text">
    <string>Metallic</string>
   </property>
  </action>
  <action name="actionRoughness_Save">
   <property name="text">
    <string>Roughness</string>
   </property>
  </action>
  <action name="actionAmbient_Occlusion_Save">
   <property name="text">
    <string>Ambient Occlusion</string>
   </property>
  </action>
  <action name="actionEmissive_Save">
   <property name="text">
    <string>Emissive</string>
   </property>
  </action>
  <action name="actionDisplacement_Save">
   <property name="text">
    <string>Displacement</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GLwidget</class>
   <extends>QOpenGLWidget</extends>
   <header>glwidget.h</header>
  </customwidget>
  <customwidget>
   <class>BrushColorButton</class>
   <extends>QToolButton</extends>
   <header>buttons/brushcolorbutton.h</header>
  </customwidget>
  <customwidget>
   <class>BrushSizeButton</class>
   <extends>QToolButton</extends>
   <header>buttons/brushsizebutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
